docs/src/modules/concepts/pages/concepts.adoc

```diff
@@ -15,18 +15,62 @@ Composable:: Enables system designs to evolve from simple to complex.
 Batteries-included:: Includes orchestration, runtimes, and distributed state management.
 Production-ready:: No code changes required to move from development to production.
 
-== Anatomy of an autonomous agentic AI system
+== Anatomy of an Agentic system
 
-An autonomous agentic AI system consists of the following behaviors:
+An agentic system is a distributed system that requires a variety of behaviors and infrastructure.
 
-[horizontal]
-Agents:: Active components that pursue goals by observing input, making decisions, and taking action. May be stateful or stateless. Often orchestrate workflows, interact with memory, or invoke tools.
-Tools:: External systems, APIs, or functions that agents can call to perform tasks beyond their own capabilities, such as querying a database or triggering an external service.
-Goals:: Define the desired outcome or intent of an agent. Goals guide reasoning and determine what success looks like for a given task or interaction.
-Guardians:: Enforce constraints or boundaries on behavior. Guardians ensure the agent stays within acceptable, ethical, or operational limits, even as it explores solutions autonomously.
-Adaptation:: Enables the agent to adjust its strategy in response to changing inputs, failure states, or unexpected outcomes. A hallmark of resilient systems.
-Coordination:: Allows agents to interact with other agents, users, tools, or services in a coherent way. Coordination is essential for sequencing actions, sharing context, or achieving group-level outcomes.
-Shared State (memory):: Provides continuity across invocations, sessions, or agents. Shared state allows the system to retain knowledge, track decisions, and synchronize behavior in distributed environments.
+image:concepts:agentic-ai-system-anatomy.png[Akka Agentic Platform]
+
+[cols="1,3", options="header"]
+|===
+|Aspect |AI Role and Responsibility
+
+|Agents
+|Components that integrate with AI to perceive their environment, make decisions and take actions toward a specific goal
+
+You implement agents in Akka with the xref:java:agents.adoc[Agents] component.
+
+|Tools
+|Any public method made available to Model Context Protocol (MCP) clients.
```

Comment:
This is misleading, or unclear. Tools can either be local methods or provided by remote MCP servers.